library/core/src/slice/mod.rs

```diff
@@ -751,7 +752,7 @@ impl<T> [T] {
     #[stable(feature = "rust1", since = "1.0.0")]
     #[inline]
     pub fn windows(&self, size: usize) -> Windows<'_, T> {
-        assert_ne!(size, 0);
+        let size = NonZeroUsize::new(size).expect("size is zero");
```

Comment:
Do we have guideline about what the phrase used in expect call ?
```suggestion
        let size = NonZeroUsize::new(size).expect("`size` cannot be zero");
```

Reply:
I am not sure, but before my change it would be `assertion failed: size != 0`

Reply:
this is bike-shredding so don't worry much about this.